example.env

```
@@ -19,3 +19,6 @@ INVESTMENT_PAYEE_NAME="Investment"
 INVESTMENT_CATEGORY_GROUP_NAME="Income"
 # optional, name of the category for added investment tracking transactions
 INVESTMENT_CATEGORY_NAME="Investment"
+
+#optional, RentCast API key for fetching property data
+#RENTCAST_API_KEY="your_rentcast_api_key_here"
```

I'm a fan of leaving this uncommented.  Either way, to use the new feature they would have to edit this line so that just saves them having to uncomment it.

```suggestion
RENTCAST_API_KEY="<Rentcast API key>"
```